scripts/lint-changelog.js

```diff
@@ -0,0 +1,100 @@
+"use strict";
+const fs = require("fs");
+const path = require("path");
+
+const CHANGELOG_DIR = "changelog_unreleased";
+const TEMPLATE_FILE = "TEMPLATE.md";
+const CHANGELOG_CATEGORIES = [
+  "angular",
+  "api",
+  "cli",
+  "css",
+  "flow",
+  "graphql",
+  "handlebars",
+  "html",
+  "javascript",
+  "json",
+  "less",
+  "lwc",
+  "markdown",
+  "mdx",
+  "scss",
+  "typescript",
+  "vue",
+  "yaml"
+];
+const CHANGELOG_ROOT = path.join(__dirname, `../${CHANGELOG_DIR}`);
+const showErrorMessage = message => {
+  console.error(message);
+  process.exitCode = 1;
+};
+
+const files = fs.readdirSync(CHANGELOG_ROOT);
+for (const file of files) {
+  if (file !== TEMPLATE_FILE && !CHANGELOG_CATEGORIES.includes(file)) {
+    showErrorMessage(`Please remove "${file}" from "${CHANGELOG_DIR}".`);
+  }
+}
+for (const file of [TEMPLATE_FILE, ...CHANGELOG_CATEGORIES]) {
+  if (!files.includes(file)) {
+    showErrorMessage(`Please don't remove "${file}" from "${CHANGELOG_DIR}".`);
+  }
+}
+
+const authorRegex = /by \[@(.*?)\]\(https:\/\/github\.com\/\1\)/;
+
+const template = fs.readFileSync(
+  path.join(CHANGELOG_ROOT, TEMPLATE_FILE),
+  "utf8"
+);
+const templateComment = template.match(/<!--[\s\S]*?-->/);
```

Comment:
It still works because include converts arrays to string, but you forgot `[0]`.

Reply:
fixed https://github.com/prettier/prettier/pull/7320/commits/a6fa99696b76e840e8e8fe40093457b4e40b8e6b